test/integration/setup.ts

```diff
@@ -1,32 +1,32 @@
 import { ethers } from "ethers";
-import Web3 from "web3";
+import { OpenSeaSDK } from "../../src/sdk";
+import { Network } from "../../src/types";
 import {
-  ALCHEMY_API_KEY,
   MAINNET_API_KEY,
+  RPC_PROVIDER,
   WALLET_PRIV_KEY,
-} from "../__tests__/constants";
-import { OpenSeaSDK } from "../sdk";
-import { Network } from "../types";
+} from "../utils/constants";
+
+for (const envVar of [
+  "WALLET_PRIV_KEY",
+  "SELL_ORDER_CONTRACT_ADDRESS",
+  "SELL_ORDER_TOKEN_ID",
```

Comment:
Do we want to force someone to hold a token to run any of the integration tests? I could see someone wanting to run tests but not having a token to sell in the wallet. This would prevent them from running the other tests.

Reply:
sure ill remove, itll just skip that test now